src/commands/base-command.ts

```diff
@@ -7,7 +7,6 @@ import { format } from 'util'
 
 import { DefaultLogger, Project } from '@netlify/build-info'
 import { NodeFS, NoopLogger } from '@netlify/build-info/node'
-// @ts-expect-error TS(7016) FIXME: Could not find a declaration file for module '@net... Remove this comment to see the full error message
 import { resolveConfig } from '@netlify/config'
```

Comment:
This is fixed by this:
https://github.com/netlify/build/pull/5692/commits/cf459316c28d49444330593fcc393c0febc80cb0